test/integration/scheduler/plugins/plugins_test.go

```diff
@@ -1338,9 +1338,8 @@ func TestUnReservePreBindPlugins(t *testing.T) {
 				t.Errorf("Error while creating a test pod: %v", err)
 			}
 
-			if test.fail {
+			if test.reject {
```

Comment:
it should have been `wantReject`, but that's a nit.

Reply:
done